src/Sentry/ISessionManager.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Sentry
+{
+    internal interface ISessionManager
+    {
+        Session? CurrentSession { get; }
```

Comment:
Exposing the reference will make it harder for us to synchronize things. Ideally mutations would come through method calls into this interface instead